robot/src/main/java/frc/robot/Constants.java

```diff
@@ -33,6 +34,7 @@
         public static final int kRightStickY = 5;
         public static final int kRightTrigger = 3;
 
+        public static final boolean kInvertMoveSpeed = true;
```

Comment:
This shouldn't be a property of the Gamepad, but instead of the drivetrain

Reply:
I think this depends on which is providing the inverted value. Is the controller providing a negative value to move forward or is the drivetrain receiving a negative value and moving forward.

If it's the former then negating in the drivetrain will become problematic for autonomous commands. If it's the latter then I agree negating in the drivetrain is correct.

Reply:
I think that historically negative controller values = forward/counter clockwise turn. I'm open to whichever, as long as we're consistent. (Also I think some autonomous pathfinding libs have the same convention of -ve = forward)

Reply:
I can add the invert to both the game pad and the drivetrain, and we can play with the values when we do autonomous driving

Reply:
If we're going down that route, then can I suggest that we add flags for inverting every axis? And name the axis accordingly (e.g. leftX, leftY, etc)